lkvt.go

```diff
@@ -261,23 +319,43 @@ func (conf *config) execute(c int, ran []uint32, wg *sync.WaitGroup) {
 			randVal:   toByteArray(ran[i]),
 			count:     uint32((*conf.amount / *conf.concurrency)*c + i),
 		}
+		etcd := "etcd"
```

Comment:
on line 323, use the calculation to determine the operation type and store the type in the kv object.